ui/pages/confirmations/hooks/useAlerts.test.ts

```diff
@@ -0,0 +1,58 @@
+import { Severity } from '../../../helpers/constants/design-system';
+import { renderHookWithProvider } from '../../../../test/lib/render-helpers';
+import useAlerts from './useAlerts';
+
+describe('useAlerts', () => {
+  const ownerIdMock = '123';
+  const fromAlertKeyMock = 'from';
+  const dataAlertKeyMock = 'data';
+  const alertsMock = [
+    {
+      key: fromAlertKeyMock,
+      field: fromAlertKeyMock,
+      severity: Severity.Danger,
+      message: 'Alert 1',
+    },
+    { key: dataAlertKeyMock, severity: Severity.Warning, message: 'Alert 2' },
+  ];
+
+  const mockState = {
+    confirmAlerts: {
+      alerts: { [ownerIdMock]: alertsMock },
+      confirmed: {
+        [ownerIdMock]: { [fromAlertKeyMock]: true, [dataAlertKeyMock]: false },
+      },
+    },
+  };
+
+  describe('returns the correct', () => {
```

Comment:
Rather than having a top level describe, should we instead have a describe per property and function returned by the hook so we can have multiple tests for each one?